LuaMenu/widgets/chobby/components/chat_windows.lua

```diff
@@ -426,9 +441,9 @@ function ChatWindows:ProcessChat(chanName, userName, message, msgDate, notifyCol
 		self.activeUnreadMessages = self.activeUnreadMessages + 1
 	end
 	local nameColor
```

Comment:
this stuff belongs to c# code to filter out the stuff for everyone by admins. Server already has ignore lists and ignore features.